src/routing/types.ts

```diff
@@ -0,0 +1,13 @@
+export interface RouterProps {
+  children: Array<PageType>;
+  basename: string;
+}
+
+export interface PageType extends React.Component {
```

Comment:
This isn't used by `Page`.

Reply:
Yes, but it's used by the `Router` as the type of its children.

Reply:
You should be able to `import Page from './page'` and use that _as_ the type. :mage_man:

With this disconnect, we can change the _actual_ page type without changing this interface and it'll lead to problems.